docker/wordrank_install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Please remove this script from PR. You already download it with the wordrank repo. After it, use `awk` or `sed` to replace  `#export CC=icc CXX=icc` to `export CC=gcc CXX=g++`